src/adapters/discord.js

```diff
@@ -1,142 +1,112 @@
 const Discord = require('discord.js');
 const debug = require('debug')('bot:adapter:discord');
 
-const isString = require('lodash/isString');
-const isObject = require('lodash/isObject');
-const isArray = require('lodash/isArray');
-const isEmpty = require('lodash/isEmpty');
+const merge = require('lodash/merge');
+const pick = require('lodash/pick');
 
 // const { PREFIX } = require('./../config');
 const { discord: discordCfg } = require('../config');
 
 const discordAdapter = () => {};
 
-discordAdapter.__INIT__ = function ({ process }) {
+discordAdapter.__INIT__ = function (ctx) {
     const discordBot = new Discord.Client();
 
     discordBot.on('ready', () => {
         debug('Discrodjs ready');
     });
 
-    discordBot.on('message', (msg) => {
+    const handler = async (output) => {
+        const { message, to, reactions } = output;
+        const [, channelId, msgId] = to;
+        let { embed } = output;
+
+        const channel = discordBot.channels.find(ch => ch.id === channelId);
+
+        if (msgId && reactions) {
+            const msg = await channel.fetchMessage(msgId);
+
+            await reactions.reduce(
+                (prev, reaction) => prev.then(() => msg.react(reaction).catch((e) => {
+                    console.error(e.message);
+                })),
+                Promise.resolve(),
+            );
+        }
+
+        if (embed) {
+            const {
+                title,
+                description,
+                url,
+                fields = [],
+            } = embed;
+
+            embed = new Discord.RichEmbed();
+            embed.setColor(discordCfg.color);
+
+            if (title) {
+                embed.setTitle(title);
+            }
+
+            if (description) {
+                embed.setDescription(description);
+            }
+
+            if (url) {
+                embed.setURL(url);
+            }
+
+            fields.forEach((field) => {
+                embed.addField(field);
+            });
+
+            return channel.send({ embed }).catch((e) => {
```

Comment:
we can't send an array of outputs now, right?
upd: https://github.com/uz0/u.dcvs.ru/pull/84/files#diff-bd9c9dcd314f2d7df52935b3a6a4d504R40 for multiple, got it